### PR TITLE
Copy data explorer code

### PR DESCRIPTION
**Pull request by samuelhwilliams**
### Description of change
This patch pulls in the codebase for the Data Explorer from
https://github.com/uktrade/data-explorer/, making the minimum set of
changes required in order to get things running. This makes no
intentional changes to the resulting functionality of that code, which
may differ (e.g. single-tenant vs multi-tenant), unless required to run
(e.g. auth).

### Checklist

* [x] Have tests been added to cover any changes?
* [x] Has the [CHANGELOG](https://github.com/uktrade/data-workspace/blob/master/CHANGELOG.md) been updated?

### CODE REVIEW

File: dataworkspace/dataworkspace/apps/explorer/scripts/compile_assets.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+source ./dataworkspace/dataworkspace/apps/explorer/scripts/functions.sh
```
Comment (kristof-bb) [inline dataworkspace/dataworkspace/apps/explorer/scripts/compile_assets.sh:3]: had a small issue with this path as the package.json is under ./dataworkspace i had to change this to ./dataworkspace/apps...

**Review by kristof-bb — COMMENTED** [review]
Tested locally and seems to run fine. Nice!

File: dataworkspace/dataworkspace/apps/explorer/scripts/compile_assets.sh (hunk 1 above)
Comment (samuelhwilliams) [reply dataworkspace/dataworkspace/apps/explorer/scripts/compile_assets.sh:3]: Ah thanks - I think I intend to rip these scripts out as part of the later cleanup, but good to know. Will fix for now. 👍

**Comment by samuelhwilliams** [comment]
@michalc have got the selenium tests integrated now, if you want to have a look at that part in particular.

**Pull request merged**